parsl/executors/radical/executor.py

```diff
@@ -0,0 +1,583 @@
+"""RadicalPilotExecutor builds on the RADICAL-Pilot/Parsl
+"""
+import os
+import sys
+import time
+import parsl
+import queue
+import logging
+import inspect
+import requests
+import typeguard
+import threading as mt
+
+import radical.pilot as rp
+import radical.utils as ru
+
+from functools import partial
+from typing import Optional, Dict
+from pathlib import Path, PosixPath
+from concurrent.futures import Future
+
+from parsl.app.python import timeout
+from .rpex_resources import ResourceConfig
+from parsl.data_provider.files import File
+from parsl.utils import RepresentationMixin
+from parsl.app.errors import BashExitFailure
+from parsl.executors.base import ParslExecutor
+from parsl.app.errors import RemoteExceptionWrapper
+from parsl.serialize import pack_apply_message, deserialize
+from parsl.serialize.errors import SerializationError, DeserializationError
+
+RPEX = 'RPEX'
+BASH = 'bash'
+PYTHON = 'python'
+
+os.environ["RADICAL_REPORT"] = "False"
```

Comment:
I'm a bit uncomfortable with screwing with the environment just because someone did "import parsl" and then didn't use radical pilot at all: this is something that affects non-radical parsl users.

Reply:
That should have been addressed by us.  A [`radical.utils` PR](https://github.com/radical-cybertools/radical.utils/pull/395) now changes the default behavior.